### ISSUE TITLE
Flame Wreath still triggers randomly

### ISSUE THREAD

**Issue by Anonyxx** [Atlantiss/NetherwingBugtracker#4893]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: 

Flame wreath still triggers randomly, even when nobody moves. This is post the recent fix to flame wreath.

**Current behaviour**:

Flame wreath triggers randomly sometimes at the end of the flame wreath cast.

https://youtu.be/mt3YkH4Yt7Q?t=21969 You can see the player does not move and yet triggers flame wreath at the end of the timer.

**Expected behaviour**: 

Flame wreath should trigger on movement.

**Server Revision**: 
2770

**Comment by Wolffenstein** [comment]
It's a tough one since I cannot replicate it on my local realm - must be due to some player spell interaction.  I will add some additional checks and debug.

**Comment by Jalapan** [comment]
@Wolffenstein some of it seems related to a pet dying and sometimes its randomly

**Comment by Zarant** [comment]
Some spells like mind flay and steady shot turns your character around, it might be an issue with that. Also issuing a stay command to a pet causes a brief convulsion, possibly triggering it.

**Comment by Anonyxx** [comment]
@Zarant You can see that doesn't apply here.

1) The shaman triggers it - he has no pet out and does not move / turn.
2) The shadow priest that DOES mind flay is unaffected by flame wreath and does not turn, same with the hunter.

This appears to be entirely random and not attributable to any player interaction. The one thing I do notice is that this triggers right as the flame wreath debuff is expiring - perhaps an issue with timers checking for the debuff being off slightly?

**Comment by towneh** [comment]
Confirmed we did not experience flame wreath being triggered last night during the Shade of Aran encounter. We had a hunter which had their pet our, set to passive and was told to stay, this did not trigger it. We also had an enhancement shaman present dealing damage to the boss and interrupting one of the spell casts (the other was a rogue). I was also mind flaying throughout and this also did not trigger anything.
